### DISCUSSION TITLE
Importing fonts

### DISCUSSION THREAD

**Discussion by vaavaav** [typst/typst#124] [Q&A]
Is it possible to import fonts, just like in latex (with lualatex or xetex ...)? E.g.:

```tex
\setmainfont[
    Path=fonts/,
    Extension=.ttf,
    Ligatures=TeX,
    AutoFakeSlant=0.20,
    BoldFont=NewsGotTBold,
    FontFace={l}{n}{NewsGotTLight}
]{NewsGotT}
```

**Comment by pbsds** [comment]
Access to all fonts in CTAN would be great.

**Reply by ondohotola** [reply]
On the Mac you can import the fonts via `fontbook` which would make them available to Typst (cli)

**Reply by user202729** [reply]
"All fonts" would be difficult because currently Typst only support opentype/truetype font I believe and not metafont or type-1 font etc..

**Comment by user202729** [comment]
At least for OTF/TTF font I think #270 would solve this.
